undertow/src/main/java/org/wildfly/extension/undertow/filters/ModClusterDefinition.java

```diff
@@ -325,26 +335,72 @@ public HttpHandler createHttpHandler(Predicate predicate, ModelNode model, HttpH
 
     }
 
+    @Override
+    public void registerOperations(ManagementResourceRegistration parent) {
+        ResourceDescriptor descriptor = new ResourceDescriptor(this.getResourceDescriptionResolver())
+                .addCapabilities(Capability.class)
+                .addRequiredSingletonChildren(SingleAffinityResourceDefinition.PATH)
+                ;
+
+        for (AttributeDefinition attribute : ATTRIBUTES) {
```

Comment:
Create an enumeration for the attributes, and eliminate the ATTRIBUTES variable.